mkdocs.yml

```diff
@@ -102,7 +102,7 @@ nav:
 
     - "Release notes" :
         - "Overview"                                                             : openj9_releases.md
-        - "Version 0.33.0"                                                       : version0.33.md
+        - "Version 0.33.0, .1"                                                   : version0.33.md
```

Comment:
Hmm, this is not a very readable format and totally different from any previous release. For 15.1 and 18.1, we just changed the text to the point release. But if this situation is different and you really want the text to cover both releases, I suggest "Version 0.33.x"

(Perhaps in the future we should rename all the releases in the table of contents to the first 2 sets of digits, like the release notes in the code repo.)

Reply:
I didn't want to add 0.33.x in case there is another release for which we want different release notes, but I suppose that is very unlikely. I thought about adding two entries, one for 0.33.1 and one for 0.33.0 which both refer to the same web page, but in end used 0.33.x.